e2e/pages/enterChildren/enterChildren.js

```diff
@@ -73,7 +74,7 @@ module.exports = {
 
   enterAddress(address) {
     within(this.fields().situation.addressOfChild, () => {
-      postcodeLookup.lookupPostcode(address);
+      postcodeLookup.enterAddressManually(address);
```

Comment:
is this still broken using the old method and an api key from aat?

Reply:
Yes

Reply:
I don't believe you, shall send you the API key tomorrow